### PR TITLE
Remove Ansi Color Codes from output in tests

### PR DESCRIPTION
**Pull request by BrennanConroy**

### CODE REVIEW

File: test/Microsoft.Framework.CommonTestUtils/TestUtils.cs [hunk 1]
```diff
@@ -386,5 +386,30 @@ private static bool IsWindows()
             return PlatformHelper.IsWindows;
 #endif
         }
+
+        private static string RemoveAnsiColorCodes(string text)
+        {
+            int escapeIndex = 0;
```
Comment (moozzyk) [inline test/Microsoft.Framework.CommonTestUtils/TestUtils.cs]: looks like a great place to use a regex..

**Comment by moozzyk** [comment]
:ship: :it:

File: test/Microsoft.Framework.CommonTestUtils/TestUtils.cs [hunk 2]
```diff
@@ -386,5 +386,30 @@ private static bool IsWindows()
...
+            for (; ;)
+            {
+                escapeIndex = text.IndexOf("\x1b[", escapeIndex);
+                if (escapeIndex != -1)
+                {
+                    int endIndex = escapeIndex + 3;
+                    while (endIndex != text.Length && text[endIndex] != 'm')
```
Comment (ChengTian) [inline test/Microsoft.Framework.CommonTestUtils/TestUtils.cs]: `endIndex < text.Length`? Otherwise it will throw "index out of bounds" in edge case where text has the value of `"\x1b["`

Comment (BrennanConroy) [inline test/Microsoft.Framework.CommonTestUtils/TestUtils.cs]: endIndex wont index if it equals the length of the string, so no 'out of bounds' happens

Comment (ChengTian) [inline test/Microsoft.Framework.CommonTestUtils/TestUtils.cs]: I mean, when `text` is `"\x1b["`, `endIndex` is 3, while `text.Length` is 2

File: test/Microsoft.Framework.CommonTestUtils/TestUtils.cs (hunk 1 above)
Comment (ChengTian) [inline test/Microsoft.Framework.CommonTestUtils/TestUtils.cs]: var

File: test/Microsoft.Framework.CommonTestUtils/TestUtils.cs (hunk 2 above)
Comment (BrennanConroy) [inline test/Microsoft.Framework.CommonTestUtils/TestUtils.cs]: Oh, good point

**Comment by ChengTian** [comment]
:shipit:
